### ISSUE TITLE
Make the resource list grow!

### ISSUE THREAD

**Issue by alfonsogarciacaro** [fable-compiler/community#1]
Hi there, great Fable community contributors!

I've put together a [community resource finder](https://fable.io/community/). It's very simple for now, but we can make it more sophisticated later and I hope that it helps make your contributions more visible to Fable users.

Could you please add your resources to the `public/community.json` file? You only need to add the name/title, a brief description, a link and a category.

Thanks everybody for your help!

@ncave @MangelMaxime @Zaid-Ajaj @et1975 @forki @dbrattli @cmeeren @musheddev @zanaptak @rommsen @nojaf

**Comment by cmeeren** [comment]
Should this include all manner of Fable bindings for npm packages?

**Comment by sergey-tihon** [comment]
What about using using `fable` GitHub tag? 
https://github.com/topics/fable?o=desc&s=stars

**Comment by nojaf** [comment]
Using the fable tag on GitHub also included old non-maintained projects right? Might be misleading for beginners.

Would you also list tools here? I once made a thing that convert json to Thoth decoders (https://nojaf.github.io/jason-to-thoth/), do those kinds of things also apply?

Lastly great initiative @alfonsogarciacaro ! Let's start simple and get this thing going.

**Comment by alfonsogarciacaro** [comment]
@cmeeren Yes, I think that anything that's valuable for Fable users should be there.

@sergey-tihon That's an option, but hopefully this gives a better way to select the results (for example, like @nojaf discarding old projects). I would also like that authors are responsible of keeping the info up-to-date, and they may want to link to documentation instead of Github page, for example. This also gives us the opportunity to list resources that are not in GH (like videos or blog posts).

@nojaf Yes, tools can be included. For now I added these categories, but we can extend/modify them: https://github.com/fable-compiler/community/blob/eac7d235b603dd05274dd66d1c89768067aef9fe/src/Global.fs#L6-L10

**Comment by cmeeren** [comment]
I guess bindings should go under `library`, but what about boilerplates/examples like [fable-elmish-electron-material-ui-demo](https://github.com/cmeeren/fable-elmish-electron-material-ui-demo)?

**Comment by alfonsogarciacaro** [comment]
Probably we need a new category named "Template" or "Sample" 🤔

**Comment by cmeeren** [comment]
👍 please ping me when it's added

**Comment by alfonsogarciacaro** [comment]
[Done!](https://github.com/fable-compiler/community/commit/350f3241078f262a45d5af96f9c512e3b27d78d9)

**Comment by MangelMaxime** [comment]
@alfonsogarciacaro I think we can have both categories because Fulma-demo is not really a template per se for example. What do you think?

**Comment by cmeeren** [comment]
[fable-elmish-electron-material-ui-demo](https://github.com/cmeeren/fable-elmish-electron-material-ui-demo) is also not really a template, but more like Fulma-demo.

**Comment by alfonsogarciacaro** [comment]
Ok, let's add the "Sample" category too then :)

**Comment by davedawkins** [comment]
I have a new package to add

https://github.com/davedawkins/Fable.React.Autocomplete

Is this the correct procedure to update this list?

- Fork this repository (to my own github account - Is a branch needed?)
- Update community.json / commit / push back to repository
- Create a pull request to this repository
[ - Wait until PR applied and optionally delete  my fork of this repository ? ]

Thanks

**Comment by MangelMaxime** [comment]
Hello @davedawkins yes this is the correct procedure.

You don't necessarily need a branch you can push your changes in your own master branch.
